bedrock/base/templates/includes/sticky-promo.html

```diff
@@ -0,0 +1,18 @@
+{# This Source Code Form is subject to the terms of the Mozilla Public
+ # License, v. 2.0. If a copy of the MPL was not distributed with this
+ # file, You can obtain one at http://mozilla.org/MPL/2.0/. -#}
+
+{% block page_css %}
+  {{ css_bundle('firefox_sticky_promo') }}
+{% endblock %}
```

Comment:
I think this would go on each page that includes the promo, I don't think it can go in the promo include itself.
It also might be an option to just make it a Sass partial and `@import` it on the pages that need it, rather than making a whole separate bundle.